src/Admin/PageAdmin.php

```diff
@@ -204,10 +201,7 @@ public function getBatchActions()
         return $actions;
     }
 
-    /**
-     * @param SiteManagerInterface $siteManager
-     */
-    public function setSiteManager(SiteManagerInterface $siteManager): void
```

Comment:
Why the return type declaration was removed?

Reply:
Good catch, my mistake!